### ISSUE TITLE
about image retrieval

### ISSUE THREAD

**Issue by xuxumiao777** [cvg/Hierarchical-Localization#377]
Hi, author. 
I am not sure about the image retrieval part: For example, if the image retrieval model is trained on SF-XL dataset, can I just load the trained model to perform image retrieval on Aachen dataset? or I need to retrain the image retrieval model on Aachen dataset?

**Comment by sarlinpe** [comment]
Yes image retrieval doesn't need to be retrained to be used with a new dataset.
